components/header.js

```diff
@@ -1,16 +1,16 @@
 import styled from "styled-components";
 import { useRouter } from "next/router";
+import Router from "next/router";
 import backButton from "../public/back_button.svg";
 import Image from "next/image";
 import { printSubhead } from "../library/printSubhead";
-import { getBackPath } from "../library/getBackPath";
 
 export default function Header() {
   const { pathname } = useRouter();
 
   return (
     <StyledHeader>
-      {pathname === "/" ||
+      {pathname === "/" ? "" :
```

Comment:
```suggestion
      {pathname === "/" ? false :
```
It's cleaner to provide a Boolean value in case you are at the root since you are checking for truthiness here in order to render the Title component. Every other case in lines 14 through 16 give back Boolean values. Makes sense to give back a Boolean value here as well.
The empty string works too, but it's an internal conversion via "".length which yields zero which yields false, and that's a bit of a detour.